testsuite/features/build_validation/create_bootstrap_repositories/srv_create_bootstrap_repositories.feature

```diff
@@ -105,3 +109,7 @@ Feature: Create bootstrap repositories
 @opensuse154arm_minion
   Scenario: Create the bootstrap repository for a OpenSUSE 15.4 ARM minion
     When I create the bootstrap repository for "opensuse154arm_minion" on the server
+
+@opensuse155arm_minion
+  Scenario: Create the bootstrap repository for a OpenSUSE 15.5 ARM minion
+    When I create the bootstrap repository for "opensuse155arm_minion" on the server
```

Comment:
I wonder if we have enough disk space on our CI servers.

Reply:
I increased it recently to 2 TB in main.tf's.